packages/@aws-cdk/cdk/lib/cloudformation/stack.ts

```diff
@@ -438,7 +438,7 @@ export abstract class Referenceable extends StackElement {
  *
  * We do attribute detection since we can't reliably use 'instanceof'.
  */
-function isStack(construct: Construct): construct is Stack {
+export function isStack(construct: Construct): construct is Stack {
```

Comment:
Can we make it a static method? `Stack.isStack` (like `Array.isArray`)?

Reply:
I wanted to do that, but I was worried it will conflict with `stack.isStack = true`.

I didn't test, though, so I could.

Reply:
Worries unfounded, it does work.